### PR TITLE
feat(components): add button to show a component in fullscreen

### PR DESCRIPTION
**Pull request by chaoran-chen**
### Summary

This adds a button to the top-right of the toolbar of the visualization components to show a component in fullscreen mode.

### PR Checklist

- [ ] All necessary documentation has been adapted.
- [x] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: components/src/preact/relativeGrowthAdvantage/relative-growth-advantage.tsx
Comment (fengelniederhammer) [inline components/src/preact/relativeGrowthAdvantage/relative-growth-advantage.tsx:1]: The chart keeps it's size after exiting full screen (on firefox):

grafik

**Review by fengelniederhammer — CHANGES_REQUESTED** [review]
Looks really good, except for the problem with the resizing of the relative growth advantage chart.

File: components/src/preact/relativeGrowthAdvantage/relative-growth-advantage.tsx
Comment (fengelniederhammer) [reply components/src/preact/relativeGrowthAdvantage/relative-growth-advantage.tsx:1]: @chaoran-chen could you please check whether it works in Safari now? We tried to fix the resize issue.

Comment (chaoran-chen) [reply components/src/preact/relativeGrowthAdvantage/relative-growth-advantage.tsx:1]: In Chrome (my default browser), this works well. In Safari, I just noticed that the relative growth advantage calculation doesn't work at all (also not with the existing versions). I'll create an issue but it's independent of this PR. For the other plots, the full screen button works.

**Review by fengelniederhammer — APPROVED** [review]

**Pull request merged**